src/cromshell/utilities/IOUtils.py

```diff
@@ -0,0 +1,45 @@
+from pathlib import Path
+
+from termcolor import colored
+
+
+def copy_file(source, destination):
+    source_path = Path(source)
+    destination_path = Path(destination)
+    destination_path.write_bytes(source_path.read_bytes())
```

Comment:
What happens if source doesn't exist?  

I think you probably need some error handling here.

Reply:
This was previously used for creating a backup for the submission table file but the module i'm currently using automatically creates a bakup file when editing so I'll remove this function as its no longer being used.